File: python/questao2.py
preco = float(input ("Digite o valor do produto: "))
desconto = float(input ("O desconto será de quantos %? (Digite apenas os numeros) "))


desconto = desconto/100 * preco
preco = preco - desconto

print("O desconto foi de R$", desconto, "e o novo preço é de R$", preco)

File: c/questao2.c
#include <stdio.h>

int main() {
    float preco, desconto;

    printf("Digite o valor do preco atual: ");
    scanf("%f", &preco);

    printf("Digite a porcentagem de desconto: ");
    scanf("%f", &desconto);

    desconto = preco * (desconto / 100);
    preco = preco - desconto;

    printf("O desconto no preco foi de R$%.2f\n", desconto);
    printf("O novo preco é de R$%.2f\n", preco);

    return 0;
}